libmscore/tempotext.cpp

```diff
@@ -190,6 +190,7 @@ void TempoText::textChanged()
                         }
                   }
             }
+      score()->fixTicks();
```

Comment:
Also, there seems to be a dirty check in InspectorBase,  https://github.com/musescore/MuseScore/blob/master/mscore/inspector/inspectorBase.h#L46
so that could be used instead, I'm guessing